### PR TITLE
doc: Update documentation and versions for 1.6.1 release

### PR DESCRIPTION
**Pull request by umapraseeda**
Update documentation for 1.6.1 release

Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

### CODE REVIEW

**Review by rlubos — COMMENTED** [review]
`release-notest-latest.rst` still points to `known issues for nRF Connect SDK v1.6.0`.

I assume we're still pending the actual changelog update, right?

**Comment by b-gent** [comment]
> `release-notest-latest.rst` still points to `known issues for nRF Connect SDK v1.6.0`.
> 
> I assume we're still pending the actual changelog update, right?

Updating release_notes_latest is important on the master branch (PR #5128 ). I can do the change here but the changelog is not displayed on the release version of doc anyway.

About the changelog update - the point is that there is not much to update here. The whole release is a few commits that fix two issues, unless I am missing something here. So I just put them in Highlights and in such case, we would not be needing a changelog. This needs to be discussed.

File: doc/nrf/releases/release-notes-1.6.1.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+It includes the TF-M, MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
+nRF53 Series devices are now supported for production.
+Wireless protocol stacks and libraries may indicate support for development or support for production for different series of devices based on verification and certification status.
+See the release notes and documentation for those protocols and libraries for further information.
+
+Highlights
+**********
+
```
Comment (b-gent) [inline doc/nrf/releases/release-notes-1.6.1.rst:20]: Link to 1.6.0 release notes here

**Comment by rlubos** [comment]
> About the changelog update - the point is that there is not much to update here. The whole release is a few commits that fix two issues, unless I am missing something here. So I just put them in Highlights and in such case, we would not be needing a changelog. This needs to be discussed.

Yeah I agree there's not much, I was just wondering whether it's needed to have a changelog section (with the content of the highlights) or not. But if from the documentation point of view the highlights are enough, I'm fine with that. 

@shantha-14 Do you have any opinion to this? Can we stick to highlights only?

**Review by rlubos — APPROVED** [review]
LGTM. I've grepped through the repository, and to me it seems that 1.6.1 was added in all needed places. Double checked the "known issues" section, looks good as well - fixed issues have been removed from 1.6.1.

**Comment by rlubos** [comment]
@b-gent Provided we don't bother with the changelog, is there any more work pending? Is DNM still needed?

**Comment by b-gent** [comment]
> @b-gent Provided we don't bother with the changelog, is there any more work pending? Is DNM still needed?

I'm still checking, please don't take it off yet.

**Comment by b-gent** [comment]
There is also this PR in nrfxlib that must go in and then manifest must be updated here:
https://github.com/nrfconnect/sdk-nrfxlib/pull/526

Also, for some reason this PR is not generating doc preview properly. Gerard is investigating, possible reason is because it is not targeting master. It is not a blocker because GitHub Actions does produce a doc artifact, it just cannot upload it for some reason.

**Comment by rlubos** [comment]
@

> There is also this PR in nrfxlib that must go in and then manifest must be updated here:
> [nrfconnect/sdk-nrfxlib#526](https://github.com/nrfconnect/sdk-nrfxlib/pull/526)
> 
> Also, for some reason this PR is not generating doc preview properly. Gerard is investigating, possible reason is because it is not targeting master. It is not a blocker because GitHub Actions does produce a doc artifact, it just cannot upload it for some reason.

Can you add a commit that updates the sdk-nrfxlib manifest entry to `pull/head/526`, so that we get the CI doc build result? (just in case).

**Comment by rlubos** [comment]
@b-gent 

>Conflicting files
west.yml

Rebase?

File: west.yml [hunk 2]
```diff
@@ -114,7 +114,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 8f3ef0d73318150c9eb5843eb5bf9ad7772ce750
+      revision: pull/526/head
```
Comment (rlubos) [inline west.yml:117]: Please update SHA: `c5efbc83787fa168123058e68e65c451e9c4345e`

**Comment by b-gent** [comment]
I can't see any issues in the output so when the CI passes, I think we can merge this.

**Review by b-gent — APPROVED** [review]

File: doc/nrf/known_issues.rst [hunk 3]
```diff
@@ -1087,14 +1088,14 @@ DRGN-15852: In rare cases on nRF53 Series, the scanner generates corrupted adver
   * TX_Power
   * Advertising_SID
 
-.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+.. rst-class:: v1-6-1 v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
```
Comment (ilhanates) [inline doc/nrf/known_issues.rst:1091]: this was also fixed in sdc for 1.6.1
https://github.com/nrfconnect/sdk-nrf/pull/5122
https://github.com/nrfconnect/sdk-nrfxlib/pull/517

**Pull request merged**